engine/xml-config/src/main/java/org/datacleaner/configuration/JaxbConfigurationReader.java

```diff
@@ -52,56 +52,16 @@
 import org.apache.metamodel.util.SimpleTableDef;
 import org.apache.metamodel.xml.XmlSaxTableDef;
 import org.datacleaner.api.RenderingFormat;
-import org.datacleaner.configuration.jaxb.AbstractDatastoreType;
-import org.datacleaner.configuration.jaxb.AccessDatastoreType;
-import org.datacleaner.configuration.jaxb.BerkeleyDbStorageProviderType;
-import org.datacleaner.configuration.jaxb.CassandraDatastoreType;
-import org.datacleaner.configuration.jaxb.ClasspathScannerType;
+import org.datacleaner.configuration.jaxb.*;
```

Comment:
While I do kind of "like" this different way of organizing imports, I fear that we will every time be committing back and forwards with new styles of organizing imports because we use different IDEs. I don't think eclipse has a way to do it the same way as IntelliJ does, so I guess I would like to ask you to revert to the Eclipse-way where every import is explicitly imported and